tools/gfx/debug-layer.cpp

```diff
@@ -285,6 +285,12 @@ void validateAccelerationStructureBuildInputs(
     }
 }
 
+Result DebugDevice::getNativeHandle(Desc::ExistingDeviceHandles* outHandle)
+{
+    outHandle = &Desc::ExistingDeviceHandles::createDummyHandles();
```

Comment:
Shouldn't return 0 here, instead DebugDevice is a layer on top an actual device, so you'll want to dispatch the call to the internal deviceObject, just like other methods for DebugDevice.